src/speakers/components/desktop-filters.tsx

```diff
@@ -1,113 +1,41 @@
-import React, { useCallback } from 'react';
-import classNames from 'classnames';
+import React, { useCallback, ReactElement, Dispatch, useState, useEffect } from 'react';
 
 import Box from '@material-ui/core/Box';
-import { makeStyles } from '@material-ui/core/styles';
 import MenuItem from '@material-ui/core/MenuItem';
 import Select from '@material-ui/core/Select';
 import InputBase from '@material-ui/core/InputBase';
 import InputLabel from '@material-ui/core/InputLabel';
 
 import { Streams, ALL_STREAMS } from '../../constants/app';
+import { Filter } from '../../entities/entities';
+import './desktop-filters.scss';
 
-import { useSpeakersFilterState, useSpeakersFiltersDispatch } from '../contexts/filters';
-
-const useStyles = makeStyles(() => ({
-    filterContainer: {
-        display: 'flex',
-        boxShadow: '5px 5px 20px #D1D1D1',
-        borderRadius: '10px',
-        margin: '20px 0',
-        position: 'sticky',
-        top: '70px',
-        width: '100%',
-        zIndex: '10',
-        background: '#fff',
-        padding: '15px 20px',
-    },
-    searchOptionText: {
-        fontSize: '14px',
-        lineHeight: '16px',
-    },
-    searchItemLabel: {
-        textTransform: 'uppercase',
-        color: '#000000',
-        marginRight: '10px',
-        flexShrink: '0',
-    },
-    searchByNameContainer: {
-        display: 'flex',
-        alignItems: 'center',
-        flexGrow: '1',
-        marginRight: '40px',
-    },
-    searchInputWrapper: {
-        background: '#FFFFFF',
-        border: '1px solid #EFEFEF',
-        borderRadius: '5px',
-    },
-    streamSelectInput: {
-        padding: '12px 10px',
-        minWidth: '150px',
-        textTransform: 'uppercase',
-    },
-    searchByNameInput: {
-        padding: '12px 10px',
-    },
-    dropdownItem: {
-        textTransform: 'uppercase',
-        fontSize: '14px',
-        lineHeight: '16px',
-    },
-    filterText: {
-        fontSize: '14px',
-        lineHeight: '16px',
-        color: '#6D7278',
-    },
-    '@media (max-width: 800px)': {
-        filterContainer: {
-            flexDirection: 'column',
-        },
-        searchByNameContainer: {
-            margin: '0 0 10px 0',
-        },
-    },
-}));
-
-const DesktopFilters = (): ReactElement => {
-    const classes = useStyles();
-
-    const { eventType, searchStr } = useSpeakersFilterState();
-
-    const filtersDispatch = useSpeakersFiltersDispatch();
+const DesktopFilters = ({ setFilter }: { setFilter: Dispatch<Filter> }): ReactElement => {
+    const [eventType, setEventType] = useState(ALL_STREAMS);
```

Comment:
do we really need to use state for the filter component? I think we can pass inside it function 'setFilter' or likewise from the outside, where we also have useState(INIT_STATE)